app/assets/javascripts/components/physical_infrastructures/physical-server-toolbar.js

```diff
@@ -0,0 +1,50 @@
+ManageIQ.angular.app.component('physicalServerToolbar', {
+  bindings: {
+    physicalServerId: '=?',
+  },
+  controllerAs: 'vm',
```

Comment:
Addressed this here: https://github.com/ManageIQ/manageiq-ui-classic/pull/1380/commits/c6c937d2f91f569fb65eee229308aa25f7412c41